package.json

```diff
@@ -197,12 +196,7 @@
     "typescript-strict-plugin": "^2.1.0",
     "vite": "^3.2.7",
     "vite-plugin-html": "^3.2.0",
-    "vite-plugin-pwa": "^0.13.3",
-    "workbox-cacheable-response": "^6.1.2",
-    "workbox-expiration": "^6.1.2",
-    "workbox-precaching": "^6.1.2",
-    "workbox-routing": "^6.1.2",
-    "workbox-strategies": "^6.1.2"
+    "vite-plugin-pwa": "^0.19.8"
```

Comment:
**Question:** do we still need pwa plugin?

Reply:
Good point. I kept it because we need to load `sw.js` alongside all js chunks somehow. I'll look into keeping it as a static file.